packages/watcher/src/enum.ts

```diff
@@ -0,0 +1,30 @@
+export const allowedSpaceTypes = [
```

Comment:
It is better to import all these enums definitions as-is from the `data-models` package.

```
import { enumerators } from 'stays-data-models';
```